From b23d63b853c2249c04fc5fa7589c7152944d7793 Mon Sep 17 00:00:00 2001
From: Frank Li <Frank.Li@freescale.com>
Date: Fri, 13 Feb 2015 04:58:40 +0800
Subject: [PATCH] MLK-10266-1 mxc: capture fix build error

cherry-pick some driver tree patch.
change interface funcation name.

Signed-off-by: Frank Li <Frank.Li@freescale.com>
---
 drivers/media/platform/mxc/subdev/mx6s_capture.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/mxc/subdev/mx6s_capture.c b/drivers/media/platform/mxc/subdev/mx6s_capture.c
index 8cb1c1d..8d5b06a 100644
--- a/drivers/media/platform/mxc/subdev/mx6s_capture.c
+++ b/drivers/media/platform/mxc/subdev/mx6s_capture.c
@@ -1544,7 +1544,7 @@ static int mx6sx_register_subdevs(struct mx6s_csi_dev *csi_dev)
 		port = of_get_next_child(node, NULL);
 		if (!port)
 			continue;
-		rem = v4l2_of_get_remote_port_parent(port);
+		rem = of_graph_get_remote_port_parent(port);
 		of_node_put(port);
 		if (rem == NULL) {
 			v4l2_info(&csi_dev->v4l2_dev,
-- 
1.8.0

